### ISSUE TITLE
Change SVG to buttons or links so that Accessibility is higher. Blind/ bad sighted people cannot swap to new pages

### ISSUE THREAD

**Issue by ppijn** [GeertHarmenVeenstra/BijRoel#14]
https://github.com/GeertHarmenVeenstra/BijRoel/blob/ee5de59129ce3293465722910949e495366e58ef/src/app/bij-roel/app.component.html#L8-L21

**Comment by ppijn** [comment]
either change to button or a link, or add tabindex to each svg (this creates a box around it, but a button will add the effect)
